akka-stream-tests/src/test/scala/akka/stream/scaladsl/SetupSpec.scala

```diff
@@ -135,7 +133,7 @@ class SetupSpec extends StreamSpec {
         }
         .named("my-name")
 
-      Source.empty.via(flow).runWith(Sink.head).futureValue shouldBe Some("setup-my-name")
+      Source.empty.via(flow).runWith(Sink.head).futureValue shouldBe Some("setup-setup-my-name")
```

Comment:
Ok, one "setup-" for each `NnnSetupStage`, since the test now uses add rather than replace attributes, and the name attribute is special and gets concatenated. 👍

Reply:
Exactly!